gitlab/v4/objects/__init__.py

```diff
@@ -3770,6 +3770,8 @@ def blame(self, file_path, ref, **kwargs):
 class ProjectPipelineJob(RESTObject):
     pass
 
+class ProjectPipelineBridge(RESTObject):
+    pass
```

Comment:
Could we move this down to lines 3780, just above its Manager class?

This way it will be consistent with the other patterns: object, manager, object, manager, etc.